etc/pyflyby/mandatory.py

```diff
@@ -6,5 +6,5 @@
 # e.g. ~/.pyflyby/mandatory.py.
 
 __mandatory_imports__ = [
-    'from __future__ import absolute_import, division',
+    'from __future__ import absolute_import, division, print_function',
```

Comment:
Is it OK to add `print_function` here? I did it so that I could make sure every file in pyflyby has it, but if this will break other code outside of pyflyby, I can revert it. The tidy-imports tests currently test against this, so they would need to be changed.

Reply:
Great!  Thanks a lot for making those upstream changes.
Re the etc/ contents: Nishant Udgaonkar already made changes towards making the imports database py2/py3 polyglot.  For now I suggest we keep it be polyglot.  At least that's what we want within D E Shaw.  That said, we have separate DEShaw-specific config databases (that inherit from the default one), so it's not a big deal if the etc/ contents ever conflict with D E Shaw desires. 
Let's put the print_function in pyflyby/.pyflyby instead of in pyflyby/etc/pyflyby.  That way it applies only to the pyflyby repo itself.

Reply:
I hadn't noticed that. I don't know how the file is used, but if a file with both 2 and 3 imports (and six for things that moved) works, that is fine. It wouldn't catch any imports that are gone, but if that isn't something it is supposed to do, then keeping a single file is the simplest.

How do I add the print_function to .pyflyby? If I understand the code correctly, there are no default filenames for mandatory imports https://github.com/pyflyby/pyflyby/blob/8e3dba15454328ea26ea09a78b98bea3dad58742/lib/python/pyflyby/_importdb.py#L287.

Reply:
The older style was that functionality was based on the filename (base name of the path).  The newer style is to have things like __remove__ or __mandatory_imports__ in a file of any name.  So we just need to put this into the .pyflyby file:

```
__mandatory_imports__ = [
    'from __future__ import print_function',
]

```

Reply:
Thanks. I thought I had tried that, but I guess I did it wrong, because it didn't work for me. But this works for me now. I suppose we should edit setup.py collect_imports script that generates the file. Currently it is broken because of https://github.com/pyflyby/pyflyby/issues/12.